performance/scaling_data_ingestion.rst

```diff
@@ -115,7 +115,13 @@ From a throughput standpoint, you can expect data ingest ratios of 250K - 2M row
 
 .. note::
 
-    To avoid opening too many connections to worker nodes, we recommend running only two COPY commands on a distributed table at a time. In practice, running more than four at a time rarely results in performance benefits. An exception is when all the data in the ingested file has a specific partition key value, which goes into a single shard. COPY will only open connections to shards when necessary.
+  Make sure your benchmarking setup is well configured so you can observe optimal COPY performance. Follow these tips:
+
+  * We recommend a large batch size (~ 50000-100000). You can benchmark with multiple files (1, 10, 1000, 10000 etc), each of that batch size.
+  * Use parallel ingestion. Increase the number of threads/ingestors to 2, 4, 8, 16 and run benchmarks.
+  * Use a compute-optimized coordinator, 8-16 vcpus. For the workers choose memory-optimized boxes with a decent number of vcpus (4-8).
```

Comment:
Can we not specify the actual numbers 8-16 vcpus and 4-8 etc, just because hardware differs from usecase to usecase.

Reply:
Just leave it at compute-optimized for coordinator and memory-optimized for workers?